web/cobrands/tfl/assets.js

```diff
@@ -23,7 +23,20 @@ var defaults = {
     body: "TfL"
 };
 
-fixmystreet.assets.add($.extend(true, {}, defaults, {
+var asset_defaults = $.extend(true, {}, defaults, {
+    select_action: true,
+    no_asset_msg_id: '#js-not-an-asset',
+    actions: {
+        asset_found: function() {
+            fixmystreet.message_controller.asset_found();
+        },
+        asset_not_found: function() {
```

Comment:
Annoying that you can't do the same here, I think you could if asset_not_found was using this instead of an argument and argument calling fixed up.